### DISCUSSION TITLE
大家好，Q技能的能量条一直是满的，要如何切换回正常模式

### DISCUSSION THREAD

**Discussion by xiongmazi** [Melledy/Grasscutter#1273] [General]
满能量导致元素充能变得没有意义

**Comment by zhaodice** [comment]
https://github.com/Grasscutters/Grasscutter/pull/1186#issuecomment-1146532784

**Comment by jia19970607** [comment]
修改配置文件

**Comment by jia19970607** [comment]
![image](https://user-images.githubusercontent.com/105071193/178938252-213742a3-7e24-41a6-b63f-fcaa17048e66.png)

**Comment by ChisatoNishikigi73** [comment]
In config.json
